src/Symfony/Component/Form/Extension/Core/Type/TimezoneType.php

```diff
@@ -40,19 +42,41 @@ public function buildForm(FormBuilderInterface $builder, array $options)
     public function configureOptions(OptionsResolver $resolver)
     {
         $resolver->setDefaults([
+            'intl' => false,
```

Comment:
Can this default value be automatically detected depending on whether the `Intl` component is installed or not?

BTW, I think this will break some codes that currently use the `regions` option with a value different to `\DateTimeZone::ALL` even if the `Intl` component is currently installed, isn't it?

Reply:
sf/intl is  a hard requirement of sf/form, so it's always available.

Silently switching from PHP timezones to Intl, is too big a step IMHO and might break data. I'd stick with the opt-in for now.

If anything; we should add a framework option to globally opt-in; let's see how it goes :)

> BTW, I think this will break some codes that currently use the regions option with a value different to \DateTimeZone::ALL even if the Intl component is currently installed, isn't it?

Not sure i understand ... setting the `regions` option is already deprecated. This new feature doesnt support it, and will crash as such (intl=true & regions=anything but ALL)

Reply:
> (intl=true & regions=anything but ALL)

I read too fast, that's correct :)